packages/css-data/src/shorthands.ts

```diff
@@ -936,7 +936,10 @@ const expandShorthand = function* (property: string, value: CssNode) {
       );
       yield [`${property}-width`, width ?? createIdentifier("medium")] as const;
       yield [`${property}-style`, style ?? createIdentifier("none")] as const;
-      yield [`${property}-color`, color ?? createIdentifier("auto")] as const;
+      yield [
+        `${property}-color`,
+        color ?? createIdentifier("currentcolor"),
```

Comment:
Please add comment. auto is not supported in browsers and described only in spec draft